contents/blog/measure-product-market-fit.md

```diff
@@ -0,0 +1,394 @@
+
+| **Pros**                                                                                                                 | **Cons**                                                                                                                 |
+|--------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------| 
+| <span className="text-green text-lg">✔</span> Defining an ICP is an essential step toward product-market fit             | <span className="text-red text-lg">✖</span> Not useful for ad-supported products (e.g. social networks)                  |
+| <span className="text-green text-lg">✔</span> Focuses on your most valuable users                                        | <span className="text-red text-lg">✖</span> Not useful when you're pre-revenue                                           |
+| <span className="text-green text-lg">✔</span> Easy to understand – growth good, no growth bad!                           | <span className="text-red text-lg">✖</span> Hard to track ICP automatically                                  |
+| <span className="text-green text-lg">✔</span> Won't tell you why you do or don't have product-market fit                 |                                 |
+
+## Indicator #5: Retention curve
+
+- **Type:** Lagging indicator
+- **Good for:** Any software product
+
+Retention is _the main_lagging indicator of product-market fit for any type of product. If your retention curve fattens _at any point_, it's a strong sign you have product-market fit.
+
+![retention curve](../images/blog/how-to-measure-product-market-fit/retention-curve.png)
+
+You'll want to track active users over a long(ish) period of time – weeks or months, rather than days. It's best to track events or actions where a user is experiencing the _real value_ of your product. Again, logging in does not count!
+
+For our product PostHog, an [all-in-one analytics platform](http://posthog.com/), this might look like:
+
+> Show `unique users` who performed a `discovery` for the first time in the last `12 months` and then came back to perform a `discovery` on any of the next months.
+
+It's also a good idea to track different cohorts, such as retention for users who match your ideal customer profile. This will reveal how strong your fit is for different users, and plan your roadmap accordingly.
+
+> 💡 **PostHog Tip:** It's easy to track retention in PostHog using the [retention insight type](/docs/product-analytics/retention), which you can segment by cohorts (e.g. users by job title, age, etc.). See also: [How to discover features that drive user retention](/tutorials/feature-retention).
+
+### Who should use retention?
+
+Everyone. It's easy to understand and a reliable way to confirm product-market fit.
```

Comment:
What about a product that is only used a few times a year, like Airbnb or an online car dealership? Retention would be hard to track and not super useful.

Reply:
Could probably downgrade to almost everyone, but in both cases they'd be looking to retain properties and dealers, right? I can add some examples of what retention looks like for different types of products here.